### ISSUE TITLE
Rate Limit RX

### ISSUE THREAD

**Issue by AqsaKashaf** [libmoon/libmoon#105]
Hi,

Can I rate limit my receiver interface? I want to emulate a link like the LinkUnqueue feature in the click router. Is it possible to do that with libmoon?

**Comment by emmericp** [comment]
No, I'm not aware of any hardware that would support such a feature. Doing it in software is either pointless (dropping them explicitly, and spending effort on packets that you don't want) or trivial (fetch less often)
